### ISSUE TITLE
Impossible values got from usage impact

### ISSUE THREAD

**Issue by bpetit** [Boavizta/boaviztapi#101]
## Bug description

With :

- **hours_use_time = 1** h
- **hours_electrical_consumption = 10.153711771428572** W
- **usage_location = FRA** so **gwp_factor = 0.098** (which is in kg CO2eq per kWh according to the api documentation)

I get gwp usage impact of **0.995 kgCO2eq** in verbose USAGE-1.

## To Reproduce

Send a request with the exact same parameters, on api version 0.1.1. I'll confirm it still happens in 0.1.2

## Expected behavior

A possible value for use impact gwp.

## JSON OUTPUT

```
          "USAGE-1": {
            "unit": 1,
            "hash": 0,
            "hours_use_time": {
              "input_value": 1,
              "used_value": 1,
              "status": "UNCHANGED"
            },
            "hours_electrical_consumption": {
              "input_value": 10.153711771428572,
              "used_value": 10.153711771428572,
              "status": "UNCHANGED"
            },
            "usage_location": {
              "input_value": "FRA",
              "used_value": "FRA",
              "status": "UNCHANGED"
            },
            "gwp_factor": {
              "input_value": null,
              "used_value": 0.098,
              "status": "SET"
            },
            "pe_factor": {
              "input_value": null,
              "used_value": 11.289,
              "status": "SET"
            },
            "adp_factor": {
              "input_value": null,
              "used_value": 4.86e-8,
              "status": "SET"
            },
            "impacts": {
              "gwp": {
                "value": 0.995,
                "unit": "kgCO2eq"
              },
              "pe": {
                "value": 115,
                "unit": "MJ"
              },
              "adp": {
                "value": 4.93e-7,
                "unit": "kgSbeq"
              }
            }
          }
        }
      }
    }
  }
 ```

**Comment by bpetit** [comment]
It seems either the unit is wrong (should be gCO2eq ?) or there is a 1000 factor lost somewhere ?

**Comment by bpetit** [comment]
Seems fixed in 0.1.2, I get **0.000902**.

Closing the issue !
